### ISSUE TITLE
Infinite Loop! Stack Overflow while calling ScaleX, ScaleY etc.

### ISSUE THREAD

**Issue by Teja-Konjeti** [ZieIony/Carbon#188]
This is only after the latest commit!

**Comment by Teja-Konjeti** [comment]
I got the exception in ImageView

**Comment by ZieIony** [comment]
It's caused by improper method proxy on Android 3.+. Use the previous commit for now.

**Comment by Teja-Konjeti** [comment]
Plz correct it asap! I need to use this commit as I want to set ripple drawable color at runtime i.e., my previous issue

**Comment by Teja-Konjeti** [comment]
Quick Question: How to remove elevation when focused for FAB?

**Comment by ZieIony** [comment]
Please create new issue every time you start a new thread. If you wish to discuss more smaller things, please join us on G+: https://plus.google.com/u/0/communities/111973718340428039040

**Comment by ZieIony** [comment]
fixed
